### PR TITLE
Migrate to Circle CI 2.0 workflows

### PR DESCRIPTION
**Pull request by 0xadada**
* [x] parallelize each Ember try scenario
* [x] successful build took `4:45`, a nice improvement

Looks really nice in the Circle CI Workflows UI
screen shot 2018-02-07 at 9 27 46 am

Fixes https://github.com/Vestorly/torii/issues/402

### CODE REVIEW

**Comment by 0xadada** [comment]
@bantic 
yaay-here-you-go

**Comment by 0xadada** [comment]
This might be useful now that Circle 1 will be deprecated this coming summer

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+      - restore_cache:
+          keys:
+            - dependency-cache-{{ .Branch }}-{{ checksum "yarn.lock" }}
+            - dependency-cache-{{ .Branch }}-
+            - dependency-cache-
+      - run:
+          name: Install dependencies
+          command: |
+            yarn global add ember-cli
+            yarn global add bower
```
Comment (Dhaulagiri) [inline .circleci/config.yml]: i _think_ we can get rid of this since we don't use bower anymore

Comment (0xadada) [reply .circleci/config.yml]: @Dhaulagiri i tried that, and the build in one of the older versions blew up.

**Comment by 0xadada** [comment]
@bantic Starting Mar 15, 2019 "All projects will require a CircleCI 2.0 configuration". CircleCI 1.0 jobs will be viewable but will no longer run.  Seems like a good time to merge this PR.

[Read more](https://circleci.com/sunset1-0/eol-policy/)

**Comment by 0xadada** [comment]
tagged @mixonic

**Comment by Dhaulagiri** [comment]
Thanks @0xadada !

**Pull request merged**